### PR TITLE
Implement Config::addRule and Config::addRules methods

### LINKED ISSUE #1428: Config::addRules and Config::addRule
Could be useful to have fixers addition methods on Config class.

In some case, we could import config from a bridge like this: https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge

And add some fixers like header_comment.

Actually, with 2.x, we have to do this:

```php
use SLLH\StyleCIBridge\ConfigBridge;

$header = 'test';

$config = ConfigBridge::create();

return $config
    ->setUsingCache(true)
    ->setRiskyAllowed(true)
    ->setRules(array_merge($config->getRules(), ['header_comment' => ['header' => $header]]))
;
```

Would be great if we could do this:

```php
return ConfigBridge::create()
    ->setUsingCache(true)
    ->setRiskyAllowed(true)
    ->addRules(['header_comment' => ['header' => $header]])
;
```

Or, even better, this:

```php
return ConfigBridge::create()
    ->setUsingCache(true)
    ->setRiskyAllowed(true)
    ->addRule('header_comment', ['header' => $header])
;
```

I can provide a PR if interested.

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Closes #1428.

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
AppVeyor needs to be re-launched.

File: Symfony/CS/Tests/Config/ConfigTest.php [hunk 1]
```diff
@@ -79,4 +79,42 @@ public function testThatMutatorHasFluentInterface()
...
+
+    public function testAddRule()
+    {
+        $config = new Config();
+        $config->setRules(array(
+            'foo' => true,
+            'bar' => true,
+        ));
+
+        $config->addRule('foo', false);
```
Comment (keradus) [inline Symfony/CS/Tests/Config/ConfigTest.php]: And this test prove that this method is not to add rule but mutate it as well.
So this is de facto like `removeRule`.

**Comment by keradus** [comment]
:-1:

File: Symfony/CS/Tests/Config/ConfigTest.php (hunk 1 above)
Comment (Soullivaneuh) [inline Symfony/CS/Tests/Config/ConfigTest.php]: I can throw an exception instead if you want.

If not, I let your the liberty to close this PR and the related issue.

**Comment by GrahamCampbell** [comment]
:-1: too, sorry.

**Comment by keradus** [comment]
Closing due to discussion of #1428.

**Pull request closed without merging**